content/en-us/scripting/events.md

```diff
@@ -68,7 +68,7 @@ connection = part.Touched:Connect(onPartTouched)
 
 <h3>Waiting for Events to Fire</h3>
 
-If you want a script to yield or pause until a specific event fires, use the `Global.RobloxGlobals.wait()|Wait` function.
+If you want a script to yield or pause until a specific event fires, use the `Wait()` function.
```

Comment:
Do you mean to use `task.wait()` here too?

```suggestion
If you want a script to yield or pause until a specific event fires, use the `Library.task.wait()` function.
```

Reply:
Yup! i seems to forget this one! sorry!

Reply:
The wait is not for `task.wait` but rather, RBXSignal:Wait()